app/views/beer/beerevent.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title></title>
+</head>
+<body>
+    <div id="add event">
+        Event Type:<select >
```

Comment:
Add label tags around your labels. For example, this line could be reformatted as:
```html
<label for="event_type>Event Type:</label>
<select id="event_type">
...
```
Also, as illustrated above, you should put your id attributes on your select elements instead of the option elements. There isn't much styling that you can do for option elements and you'd want to apply styling as a group instead of individually anyhow.